include/GenEigsSolver.h

```diff
@@ -203,7 +205,7 @@ class GenEigsSolver
                 m_fac_f.noalias() = rng.random_vec(m_n);
                 // f <- f - V * V' * f, so that f is orthogonal to V
                 MapMat V(m_fac_V.data(), m_n, i); // The first i columns
-                Vector Vf = V.transpose() * m_fac_f;
+                Vf.noalias() = V.transpose() * m_fac_f;
```

Comment:
I remember I once thought about this, but the trouble is that `V` changes its size in every iteration, so `Vf.noalias()` basically has no effect.

Reply:
I see.. In this case one can preallocate the full vector and only use the needed part.
I commit this for GenEigSolver, if you like it I can do it for the other classes as well.